.github/workflows/test-coverage.yaml

```diff
@@ -0,0 +1,31 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+
+name: test-coverage
+
+jobs:
+  test-coverage:
+    runs-on: ubuntu-latest
```

Comment:
If this fails, then you can try:

```yaml
runs-on: macOS-latest
```

Sometimes there are issues because some deps  can't be installed for Ubuntu.

Reply:
How many and which tests fail? Maybe we can skip them while computing code coverage.

Reply:
And yet there is no failure on the master branch... (except for macos-latest with R-devel that is broken)

Reply:
That's weird, devtools::test() fails locally but running the failing tests individually work

Reply:
@IndrajeetPatil two of them fail, one in test-MCMglmm, the other in test-survreg